week-3/Homework/mandatory/6-lane-names.js

```diff
@@ -5,43 +5,47 @@
   
 */
 //hint: string and array methods that could be helpful (indexOf, filter)
-function getLanes() {
 
+function getLanes(arr) {
+  return arr.filter(function (el) {
+    return el.indexOf("Lane") !== -1;
+  });
```

Comment:
A slightly simpler solution would be:
function getLanes(names) {
  return names.filter((name) => name.includes("Lane"));
}